sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -593,7 +596,7 @@ export class EventHubReceiver extends LinkEntity {
    */
   isOpen(): boolean {
     const result = Boolean(this._receiver && this._receiver.isOpen());
-    log.error(
+    log.info(
```

Comment:
Below is the guideline for info channel:

> DO use the Info channel when a function operates normally.

I would imagine this mainly to refer to the function called by the user.

`isOpen()` here gets called internally only and therefore I would expect this log here to be verbose.
Otherwise, users who set the log level to info would keep seeing this one randomly pop up in the logs

Same for the logs in `_deleteFromCache()`